tests/test-tidy.sh

```diff
@@ -17,7 +17,7 @@ do
   src/libs/*|src/FreeRTOS/*) continue ;;
   *.cpp|*.h)
     echo "::group::$file"
-    clang-tidy-14 -p build "$file" || true
+    clang-tidy-16 -p build "$file" || true
```

Comment:
If clang-tidy isn't being updated, this should be undone right?

Reply:
Oh yeah, oops